cl/lib/search_utils.py

```diff
@@ -1292,10 +1293,10 @@ def clean_up_recap_document_file(item: RECAPDocument) -> None:
     """
 
     if type(item) == RECAPDocument:
-        item.filepath_local.delete()
+        await sync_to_async(item.filepath_local.delete)()
```

Comment:
`adelete()`?

Reply:
Not supported here:
```python
  File "/opt/courtlistener/cl/scrapers/tasks.py", line 269, in extract_recap_pdf_base
    response = await microservice(
    ^^^^^^^^^^^^^^^^^
  File "/opt/courtlistener/cl/lib/microservice_utils.py", line 60, in microservice
    await clean_up_recap_document_file(item)
    ^^^^^^^^^^^^^^^^^
  File "/opt/courtlistener/cl/lib/search_utils.py", line 1296, in clean_up_recap_document_file
    await item.filepath_local.adelete()
    ^^^^^^^^^^^^^^^^^
AttributeError: 'FieldFile' object has no attribute 'adelete'
```

Reply:
Well, I'm just a barrel of help, eh? So far I've learned:

 - No async in regular functions (I knew this, doh)
 - No async in DRF instances.
 - No async in file deletions.

Learning curve is swift.